.circleci/config.yml

```diff
@@ -13,11 +13,26 @@ convert_xml: &convert_xml
     name: Converting XML
     when: always
     command: |
-        mkdir ${CIRCLE_JOB}
+        mkdir -p ${CIRCLE_JOB}
         xsltproc \
           /hpx/conv.xsl Testing/`head -n 1 < Testing/TAG`/Test.xml \
           > ${CIRCLE_JOB}/Test.xml
 
+move_core_dump: &move_core_dump
+    name: Moving core dumps
+    when: on_fail
+    command: |
+        mkdir -p ${CIRCLE_JOB}
+        # Ignore errors if there are no core dumps
+        cp core.* ${CIRCLE_JOB} || true
+
+move_debug_log: &move_debug_log
+    name: Moving debug logs
+    when: on_fail
+    command: |
+        mkdir -p ${CIRCLE_JOB}
+        cp debug-log.txt ${CIRCLE_JOB} || true
```

Comment:
When interpreting this correctly, this will move the log of the last run test, right? This might not have been the test that failed, I guess.

Reply:
The logs get appended to the file. It'll contain logs from all the tests in that step. The downside of this is that the files can end up huge (GBs), but I've set it to run on failures only for that reason.

Reply:
So far just by checking what failed on CircleCI and searching for the logs of that test in the file. It has worked well enough so far. I was considering creating a log with a name corresponding to each test. I'm sure we could parse the ctest xml output to determine which tests failed but I didn't feel particularly motivated to do so. Do you think it would worth it?